polaris-ui/.env.development.local.example

```diff
@@ -48,6 +48,7 @@ REACT_APP_FEATURE_FLAG_GLOBAL_NAV="true"
 REACT_APP_FEATURE_FLAG_EXTERNAL_REDIRECT_CASE_REVIEW_APP="true"
 REACT_APP_FEATURE_FLAG_EXTERNAL_REDIRECT_BULK_UM_APP="true"
 REACT_APP_FEATURE_FLAG_BACKGROUND_PIPELINE_REFRESH="true"
+REACT_APP_FEATURE_FLAG_USED_DOCUMENT="true"
```

Comment:
you need add the new feature flag on the terraform variables aswel, may be this will give you a reference https://github.com/CPS-Innovation/Polaris/pull/1595/commits/925d501b49245129f013b5909366bcf79bc4ed57